ci/scripts/e2e-iceberg-engine-test.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Exits as soon as any line fails.
+set -euo pipefail
+
+source ci/scripts/common.sh
+
+# prepare environment
+export CONNECTOR_LIBS_PATH="./connector-node/libs"
+export AWS_REGION="us-east-1"
```

Comment:
Why `AWS_REGION` is required? IIUC the CI scripts are running on MinIO

Reply:
Because we use jdbc catalog, the underlying sdk requires aws_region even for minio.